c7n/resources/account.py

```diff
@@ -367,6 +367,52 @@ def process(self, resources, event=None):
         return []
 
 
+@filters.register('access-analyzer')
+class AccessAnalyzer(Filter):
+    """Check for access analyzers in an account
+
+    :example:
+
+    .. code-block:: yaml
+
+            policies:
+              - name: account-access-analyzer
+                resource: account
+                region: us-east-1
+                filters:
+                  - type: access-analyzer
+                    status: active
+                    trust: organization
+    """
+
+    schema = type_schema('access-analyzer',
+                        status={'oneOf': [{'enum': ['active', 'creating', 'disabled', 'failed']}]},
+                        trust={'oneOf': [{'enum': ['account', 'organization']}]})
+    schema_alias = False
+    permissions = ('access-analyzer:ListAnalyzers',)
+
+    def process(self, resources, event=None):
+        account = resources[0]
+        status = self.data.get('status', 'active')
+        trust = self.data.get('trust', '')
+
+        client = local_session(self.manager.session_factory).client('accessanalyzer')
+
+        try:
+            analyzers = self.manager.retry(client.list_analyzers)['analyzers']
+            for analyzer in analyzers:
+                if (not trust and analyzer['status'].lower() == status) or \
+                   (analyzer['type'].lower() == trust and analyzer['status'].lower()):
+                    account['c7n:AccessAnalyzer'] = analyzer
+                    return resources
+        except Exception as e:
+            self.log.warning(
+                "Exception trying to list analyzers in account: %s error: %s",
+                self.manager.config.account_id, e)
+            raise e
```

Comment:
use `raise` instead to avoid obscuring the stack trace. its actually unclear what the utility of the catch here at all is, the default error handler will also log. i'd suggest removing the try/except block.